src/flint/fmpz.jl

```diff
@@ -1284,10 +1288,22 @@ function primorial(x::Int)
     x < 0 && throw(DomainError("Argument must be non-negative: $x"))
     z = fmpz()
     ccall((:fmpz_primorial, :libflint), Nothing,
-          (Ref{fmpz}, UInt), z, x)
-    return z
+          (Ref{fmpz}, UInt), z, UInt(x))
+    return Int(z)
 end
 
+@doc Markdown.doc"""
+    primorial(x::fmpz)
+>  Return the primorial of $x$, i.e. the product of all primes less than or
+> equal to $x$. If $x < 0$ we throw a `DomainError()`.
+"""
+function primorial(x::fmpz)
+    x < 0 && throw(DomainError("Argument must be non-negative: $x"))
+    z = fmpz()
+    ccall((:fmpz_primorial, :libflint), Nothing,
+          (Ref{fmpz}, UInt), z, UInt(x))
+    return z                                                                        end
```

Comment:
`end` in the wrong place :)

Reply:
Argh, my editor is useless. I can't cut and paste without this happening. Developing code on Windows is a nightmare since Microsoft crippled the editor.

Reply:
Ouch! sorry!